pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -195,7 +196,7 @@ func (this *SnapshotManager) DeleteSnapshot(peID astrolabe.ProtectedEntityID) er
 	this.Infof("Step 2: Deleting the durable snapshot, %s, from s3", peID.String())
 	err = this.DeleteProtectedEntitySnapshot(peID, true)
 	if err != nil {
-		this.Errorf("Failed to delete the durable snapshot")
+		this.Errorf("Failed to delete the durable snapshot: %v", err)
```

Comment:
What PEID/snapshot ID were we unable to delete?

Reply:
Same to the above.